shark-graph/src/test/java/shark/LongScatterSetTest.kt

```diff
@@ -0,0 +1,128 @@
+package shark
+
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import shark.LongScatterSetAssertion.Companion.assertThat
+import shark.internal.hppc.HHPC
+import shark.internal.hppc.LongScatterSet
+
+class LongScatterSetTest {
+
+  @Test fun `verify LongScatterSet#add operation works correctly`() {
+    val set = LongScatterSet()
+    verifyAddOperation(set)
+  }
+
+  @Test fun `verify LongScatterSet#remove operation works correctly`() {
+    val set = LongScatterSet()
+    verifyRemoveOperation(set)
+  }
+
+  @Test fun `verify LongScatterSet#release operation works correctly`() {
+    val set = LongScatterSet()
+    set.add(42)
+    set.release()
+
+    assertThat(set)
+        .isEmpty()
+        .doesNotContain(42)
+  }
+
+  @Test fun `verify LongScatterSet#ensureCapacity does not break other operations`() {
+    val setForAdditionOperationCheck = LongScatterSet().apply { ensureCapacity(10) }
+    val setForRemoveOperationCheck = LongScatterSet().apply { ensureCapacity(10) }
+    val setForTwoAdditionsCheck = LongScatterSet().apply {
+      add(42)
+      add(10)
+      ensureCapacity(100)
+    }
+
+    // Ensure capacity before adding/removing elements
+    verifyAddOperation(setForAdditionOperationCheck)
+    verifyRemoveOperation(setForRemoveOperationCheck)
```

Comment:
Same comment as before: this is the "test" part, so there shouldn't be any assertion or verification in it.

Reply:
Updated!